kpops/pipeline.py

```diff
@@ -83,6 +86,72 @@ def to_yaml(self) -> str:
             self.model_dump(mode="json", by_alias=True, exclude_none=True)["components"]
         )
 
+    def build_execution_graph_from(
+        self,
+        components: list[PipelineComponent],
+        reverse: bool,
+        runner: Callable[[PipelineComponent], Coroutine],
+    ) -> Awaitable:
+        sub_graph_nodes = self.__get_graph_nodes(components)
+
+        async def run_parallel_deployments(deployments: list[Coroutine]) -> None:
```

Comment:
an abstract name is fine, just the parameter name `tasks` wasn't appropriate when it takes coroutines